sBTC SIP

```diff
@@ -0,0 +1,166 @@
+
+- **Stacks Transaction Fee Sponsorship:** sBTC will include the option to have sBTC transactions on Stacks be sponsored in return for some sBTC. Using the approach suggested in this issue, sBTC users will be able to nearly spend sBTC as gas by getting support from an existing STX holder.
+- **Signer Wallet Rotation:** Mechanisms are provided for the scenario where a signer wants to rotate their key. For this to happen, signers must coordinate offline and vote on-chain on the new signer set (aka set of keys). Once the new signer set is determined, the signers conduct a wallet handoff and re-execute DKG.
+
+### sBTC Bootstrap Signer Eligibility
+The Stacks 2.5 epoch activated the Stacks Signer network, whereby Stackers (users who lock STX in consensus in order to earn BTC rewards) are able to participate in Stacks consensus. Stackers vote to append blocks to the Stacks blockchain by producing a signature over the block. Each Stacker's share of the signature (its weight) is proportional to the fraction of locked STX it owns. This essential function ensures the security and liveness of the Stacks network.
+
+The sBTC Bootstrap Signers will be a separate subset of Stacks Signers. For the sBTC Bootstrap release, Signers will run a binary in addition to the core Stacks software. There will be 15 signers participating in the sBTC bootstrap release. In a subsequent release and hard fork, sBTC will be enshrined in Stacks consensus and Signers will only need to maintain a single binary.
+
+The following eligibility criteria has been used to identify the sBTC Bootstrap Signers:
+- **Stacks 2.5 Participation:** Active participation running a signer on Stacks 2.5 testnet or mainnet for at least two previous cycles.
+- **Technical Performance and Uptime:** Consistency in operational status and network stability. Signers should demonstrate a target uptime of at least 99%.
+- **Communication & Availability:** Running an sBTC Signer is a pivotal role in the Stacks ecosystem, which requires a high degree of availability and communication with Stacks core developers. Signers must be able to respond to updates within 24 hours.
+- **Ecosystem Alignment:** Commitment to the growth and principles of the Stacks and Bitcoin ecosystems with contributions to advance the network.
+
+By implementing a whitelisted signer set for the sBTC Bootstrap release, this helps to reduce time to market to launch sBTC sooner, remove complexity from the initial protocol design, and ensure the sBTC protocol is secure at launch.
+
+### Activation
+sBTC Bootstrap is designed to activate on Stacks Nakamoto as defined in SIP-021. Therefore, this SIP is only meaningful when SIP-021 activates. The sBTC Working Group plans to observe at least 2-4 weeks of network behavior on Stacks Nakamoto to ensure a stable release. After this period, sBTC Bootstrap can be activated on the Stacks network without requiring a separate hard fork.
```

Comment:
It sounds like the SIP delegates the criteria for the activation of this SIP to a 3rd party?  If so, then this SIP cannot be accepted as is.  Per SIP-000, the _Steering Committee_ decides when a SIP is ratified, and uses a falsifiable, unambiguous procedure for doing so in the SIP's "Activation" section.  So, this paragraph must be dropped -- there's no way the Steering Committee can fulfill its role when its authority is outsourced.

Reply:
I don't know how to reconcile this comment with the activation process we have planned. We need to ensure Nakamoto is stable prior to activating sBTC. If this SIP is approved, but Nakamoto is delayed or is not stable, then sBTC cannot activate. This will be up to the discretion of the sBTC Working Group. Any thoughts @AshtonStephens?

Reply:
> This will be up to the discretion of the sBTC Working Group. 

The activation of this SIP, and all others, is solely up to the discretion of the steering committee in the manner SIP-000 describes -- the activation section must describe an unambiguous, falsifiable activation procedure that the SC must agree to before the SIP can be transitioned to Activation-in-Progress status.

I do not agree with outsourcing this responsibility to a working group, since working groups are not part of the SIP process and thus face no procedural accountability for their role in SIP ratification.